src/libraries/Microsoft.PowerFx.Core/Binding/BinderUtils.cs

```diff
@@ -112,12 +115,11 @@ public static void LogTelemetryForFunction(TexlFunction function, CallNode node,
         /// <returns>
         /// True if a valid overload was found, false if not.
         /// </returns>
-        internal static bool TryGetBestOverload(CheckTypesContext context, IErrorContainer errors, CallNode node, DType[] argTypes, TexlFunction[] overloads, out TexlFunction bestOverload, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap, out DType returnType)
+        internal static bool TryGetBestOverload(CheckTypesContext context, IErrorContainer errors, CallNode node, TexlNode[] args, DType[] argTypes, TexlFunction[] overloads, out TexlFunction bestOverload, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap, out DType returnType)
```

Comment:
>TexlNode[] args, [](http://example.com/codeflow?start=114&length=16)

Why are we changing this?

Reply:
Small unrelated change that was found while doing the PA Client buddy build, hits a debug assert in dataflow analysis for string interp calls. Easy/safe  enough to fix while I was in there, so I just went ahead and did it.